lib/exchange/simpleswap/simpleswap_exchange_provider.dart

```diff
@@ -188,13 +187,15 @@ class SimpleSwapExchangeProvider extends ExchangeProvider {
     final to = CryptoCurrency.fromString(toCurrency);
     final inputAddress = responseJSON['address_from'] as String;
     final expectedSendAmount = responseJSON['expected_amount'].toString();
+    final extraId = responseJSON['extra_id_from'] as String;
```

Comment:
Could be extraId optional here ? Just if there could be null it will throw exception on null, so if it's not required field better to cast to optional String